requirements/env_climada.yml

```diff
@@ -25,6 +25,7 @@ dependencies:
   - python=3.8
   - rasterio>=1.1
   - requests>=2.24
+  - salib>=1.3.0
```

Comment:
I'd move this to the pip section. In any case it will be installed through pip.

Reply:
It can be installed via conda at the moment at least.